service/src/test/java/bio/terra/profile/service/azure/AzureServiceUnitTest.java

```diff
@@ -106,9 +107,9 @@ public void getManagedApps_dedupesApps() {
     when(appService.getTenantForSubscription(subId)).thenReturn(tenantId);
 
     var offer = new AzureConfiguration.AzureApplicationOffer();
-    offer.setName("FAKE_APP_NAME");
```

Comment:
This test was using a different value for the offer key than the offer name, which was technically possible given the earlier map implementation.